code/mob/living/life/critical.dm

```diff
@@ -75,7 +75,7 @@
 				boutput(owner, "<span class='alert'>Your head [pick("feels like shit","hurts like fuck","pounds horribly","twinges with an awful pain")].</span>")
 				owner.losebreath+=10
 				owner.changeStatus("weakened", 3 SECONDS)
-		if (owner.health <= -100)
+		if (owner.health <= -100 && !(owner.reagents.has_reagent("synaptizine") && owner.reagents.has_reagent("atropine"))) // Axes deathchance if you have synaptizine & atropine
```

Comment:
I'm a little skeptical of this change, as it means an atropine + synaptizine blend gives you essentially as much time as you want to fix them up. These two chems alone are (now) fantastic medical chems for deep crit patients, this line change turns them into an absolute no-brainer for deep-crit (whereas before you'd have a race against time to get the right meds into them, this now stops the clock while you work out the rest while also being useful chems for getting them *out* of deep crit).

Not necessarily requiring a change here, just somewhat cautious. I would maybe have it apply a multiplier to the `deathchance` instead of essentially removing it?

Reply:
From what I've seen, even setting the deathchance to a tenth of its normal value still makes them die pretty quickly. Any significant random chance that makes the person you're working on die through little fault of your own has, in a way, made me feel kinda cheated. I know the deathchance thing is much better than it was before, where doing anything other than cloning a flatlining patient was a waste of time, but I wanted to add in something that a somewhat experienced medic could do to avoid this randomness. 

So, from what I feel, if something's going to alter the deathchance, it should nullify it, as opposed to just reducing it. I'm okay with removing it, though!

That is a good point though, that if someone's in medbay, chances are they're going to have synaptizine and atropine in them just as a side effect of those working with deep crit, and that those, on their own, do a good enough job getting someone out of deep crit. This would be a good place for another underused chem!